test/libscap/test_suites/userspace/event_table.cpp

```diff
@@ -167,3 +167,26 @@ TEST(event_table, check_usage_of_EC_UNKNOWN_flag) {
 		}
 	}
 }
+
+TEST(event_table, check_exit_param_names) {
```

Comment:
We should enforce the rule that the first parameter in the exit event is the return value of the syscall. Unfortunately, I've not found a great way to do that so for now we just monitor the parameters' names.

Reply:
it's a best effort check, if the name matches we are quite confident that the first parameter is a return value

Reply:
this already cached the weird case of `PPME_GENERIC_X'

Reply:
Ah i see, makes sense.
But then we are not using the check on param name at runtime anymore, so this is just to avoid that in the future a new syscall exit event is added whose first param is not a return code.

Reply:
exactly, today we are quite confident the rule is valid, this test tries to prevent future issues